invenio_communities/serializers/schemas/community.py

```diff
@@ -38,6 +54,8 @@ class CommunitySchemaV1(Schema):
     page = fields.String()
     curation_policy = fields.String()
     last_record_accepted = fields.DateTime()
+    links = fields.Function(
+            serialize=lambda c: LinksSchemaV1().dump(c)[0])
```

Comment:
The reasons the ``links`` field is being skipped is because there's no ``links`` field in a community. Can you drop by me  when you have moment.